library/base/src/main/java/org/cru/godtools/base/Settings.java

```diff
@@ -25,6 +25,7 @@
     // feature discovery
     public static final String FEATURE_LANGUAGE_SETTINGS = "languageSettings";
     public static final String FEATURE_LOGIN = "login";
+    public static final String FEATURE_ANIMATE_CARD = "animateCard";
```

Comment:
I would create 2 features, and call them "tractCardSwiped" and "tractCardClicked" (or similar).

Reply:
another option if we don't want to distinguish between swipes and clicks, is just "tractCard". I'm envisioning features are more about the feature itself and not the animation that draws attention to the feature.

So, the feature in this case is cards within tracts, how we draw attention to that is currently via a bounce animation. That could change based on the overlay tests that Ann is running with Brenton. But the feature itself is not going to change.